### DISCUSSION TITLE
How to change font in wezterm

### DISCUSSION THREAD

**Discussion by uldrim** [wezterm/wezterm#3920] [Q&A]
I am using ubuntu 22.04 and have downloaded the stable release of wezterm for it.
I have downloaded Mononoki nerd font and extracted it in ~/.fonts/ 
How do I get that font in the wezterm.
If i do wezterm ls-fonts, it shows the built in fonts only.
Thank you so much for the help.

**Comment by wez** [comment]
Don't forget to run `fc-cache -v` after installing fonts in order for font-config's cache to be updated

**Comment by jankatins** [comment]
Here is a snippet which I used on linux for nerdfonts:

```bash
 cd ${HOME}
 FONTS_FOLDER="${HOME}/.local/share/fonts"
 rm -rf JetBrainsMono.zip
 wget https://github.com/ryanoasis/nerd-fonts/releases/download/v2.1.0/JetBrainsMono.zip
 unzip -o -u JetBrainsMono.zip -d ${FONTS_FOLDER}
 fc-cache -fv
 rm JetBrainsMono.zip
```

**Reply by nekowinston** [reply]
Note that this is pretty outdated, 2.1.0 is 2+ years old, and 3.0 (which is a breaking release) are the symbols which are included in the next release / nightly.
